vxdev/update-code.sh

```diff
@@ -49,6 +53,23 @@ elif [[ $BRANCH == 'stable' ]]; then
 	cd vxsuite
 	./script/setup-dev
 	cd ..
+elif [[ $BRANCH == 'custom' ]]; then
+	read -p "Enter the branch name: " BRANCH_NAME
+	cd vxsuite
+	git checkout main
+	git pull
+	if [ !`git branch -r --list origin/$BRANCH_NAME` ]
+	then
+		echo "Invalid Branch Name. Closing in 3 seconds."
```

Comment:
Would it be better to wrap this in a loop so you don't have to restart the whole script if you make a typo?

Reply:
updated